skills/hears.js

```diff
@@ -32,7 +34,8 @@ const learningHandler = async (bot, message) => {
       await transaction(Point.knex(), async trx => {
         const messageRecord = await Message.query(trx).insertAndFetch({
           text: message.event.text,
-          datetime: moment().format(),
+          datetime: message.event.ts,
+          slack_event_id: message.event_id,
```

Comment:
Maybe also add `slack_event_id` field in `Message` model in `models/schema.js`.

Reply:
good catch, done